modules/fbs-core/web/src/app/page-components/sql-playground/sql-input-tabs/sql-input-tabs.component.ts

```diff
@@ -360,4 +364,16 @@ export class SqlInputTabsComponent
     this.submitToTask();
     this.saveToLocalStorage();
   }
+
+  private checkForCheckerConfig() {
+    this.checkerService
+      .getChecker(this.selectedCourse.id, this.selectedTask.id)
+      .subscribe((checkers) => {
+        if (checkers.length === 0) {
+          this.isCheckerEmpty = true;
+        } else {
+          this.isCheckerEmpty = false;
+        }
+      });
+  }
```

Comment:
This function is a duplicate, of the function in `task-detail.component.ts`. This should be outsourced.